src/components/notices-provider/index.js

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line no-restricted-exports
```

Comment:
This is a valid ESLint rule that I feel we should avoid disabling. Two possible solutions:

1.

```jsx
import NoticesProvider from './NoticesProvider';
export default NoticesProvider;
```

2.

```jsx
export { default as NoticesProvider } from './NoticesProvider';
```
and where it gets imported: ```import { NoticesProvider } from './NoticesProvider';```